docs/index.rst

```diff
@@ -54,6 +53,18 @@
    /rst/replaying/usage/configuration
 
 
+.. _index_tutorials:
+
+.. toctree::
+   :caption: Tutorials
+   :maxdepth: 2
+   :numbered: 5
+   :hidden:
+
+   /rst/tutorials/dynamic_types
+   /rst/tutorials/foxglove_tutorial
+
+
```

Comment:
I did this because the tutorial about configuring dynamic types in Fast-DDS applied to both. Now the specific tutorials are in their respective getting started sections, and I don't think there will be any more.